packages/services/index.js

```diff
@@ -205,6 +202,67 @@ const startRelayer = () =>
     resolve(startServer)
   })
 
+const startListener = () =>
+  new Promise(resolve => {
+    const cwd = path.resolve(__dirname, '../../infra/discovery')
+
+    const spawnedListener = spawn(
+      `node`,
+      [
+        'src/listener/listener.js',
+        '--network=localhost',
+        '--marketplace',
+        '--identity',
+        '--elasticsearch'
```

Comment:
let's also add --growth (it captures events in the Db for Origin Rewards).

Reply:
ok, will add in a separate PR